src/constants/appConstant/url.ts

```diff
@@ -0,0 +1,8 @@
+// import {githubConfig} from '../../../config/config'; // you can include client id from readme file in config
+
+import {githubConfig} from '../../../config/config.sample';
+
+export const urls = {
+  GITHUB_AUTH: `https://github.com/login/oauth/authorize?client_id=${githubConfig.clientId}`,
```

Comment:
Should we check for `clientId` to always exist? How will we know if this is undefined in future?